openstack/image/api.go

```diff
@@ -393,6 +396,7 @@ func deleteImage(context *Context, w http.ResponseWriter, r *http.Request) (APIR
 	imageID := vars["image_id"]
 
 	_, err := context.DeleteImage(imageID)
+	fmt.Println(err)
```

Comment:
@markdryan, yeah, you're right, I need to check for println before submitting the patch. Thanks fot the catch.